source/_patterns/_colors.scss

```diff
@@ -162,23 +165,27 @@ $color-brand-srf-dok-2:                       rgba(21,78,96,1);      // #154e60
 
 $color-brand-radio-srf-2-1:                   rgba(201,29,125,1);    // #C91D7D
 $color-brand-radio-srf-2-1a00:                rgba(201,29,125,0);    // #C91D7D alpha  0%
+$color-brand-radio-srf-2-1a60:                rgba(201,29,125,.6);   // #C91D7D alpha 60%
 $color-brand-radio-srf-2-2:                   rgba(140,29,96,1);     // #8c1d60
 $color-brand-radio-srf-2-2a00:                rgba(140,29,96,0);     // #8c1d60 alpha  0%
 
 $color-brand-radio-srf-3-1:                   rgba(70,70,70,1);      // #464646
 $color-brand-radio-srf-3-1a00:                rgba(70,70,70,0);      // #464646 alpha  0%
+$color-brand-radio-srf-3-1a60:                rgba(70,70,70,.6);     // #464646 alpha 60%
```

Comment:
Neue «Farben» auch in 01-global/10-colors.twig nachtragen.

Reply:
die branding colors sind generell (noch) nicht in diesem file -> separater task